<launch>

  <!-- launch joy node -->
  <node pkg="joy" type="joy_node" name="joy_node" />

  <!-- launch open ai lunar lander node -->
  <node name="lunar_lander" pkg="koopman_operator_model_learning" type="shared_control.py" output="screen" required="true">
    <param name="data_path" value="$(find koopman_operator_model_learning)/data/" />
    <param name="main_joystick" value="right" />
    <param name="inverted" value="False" />
    <param name="draw_helipad" value="False" />
    <param name="draw_goal" value="True" />
    <param name="main_engine_power" value="13.0" />
    <param name="side_engine_power" value="0.6" />
    <param name="initial_random" value="1000" />
    <param name="viewport_w" value="600" />
    <param name="viewport_h" value="400" />
    <param name="initial_pos_x" value="10.0" />
    <param name="initial_pos_y" value="13.3" />
    <param name="initial_pos_ang" value="0.0" />
    <param name="initial_goal_x" value="10.0" />
    <param name="initial_goal_y" value="6.0" />
  </node>

</launch>
